resources/assets/js/components/teams/TeamEditForm.vue

```diff
@@ -0,0 +1,324 @@
+                                id="team-visible-buttons"
+                                :is-error="$v.team.visible.$error"
+                                @input="$v.team.visible.$touch()">
+                        </custom-radio-buttons>
+                    </div>
+
+                    <label for="team-attendable-buttons" class="col-sm-2 col-form-label">
+                        <abbr title="Used for attendance tracking">Attendable</abbr><span style="color:red">*</span>
+                    </label>
+                    <div class="col-sm-10 col-lg-2">
+                        <custom-radio-buttons
+                                v-model="team.attendable"
+                                :options="yesNoOptions"
+                                id="team-attendable-buttons"
+                                :is-error="$v.team.attendable.$error"
+                                @input="$v.team.attendable.$touch()">
+                        </custom-radio-buttons>
+                    </div>
+
+                    <label for="team-self-serviceable-buttons" class="col-sm-2 col-form-label">
+                        <abbr title="Users can join/leave via self-service">Self-Serviceable</abbr><span style="color:red">*</span>
+                    </label>
+                    <div class="col-sm-10 col-lg-2">
+                        <custom-radio-buttons
+                                v-model="team.self_serviceable"
+                                :options="yesNoOptions"
+                                id="team-self-serviceable-buttons"
+                                :is-error="$v.team.self_serviceable.$error"
+                                @input="$v.team.self_serviceable.$touch()">
+                        </custom-radio-buttons>
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <button type="submit" class="btn btn-primary">Save Changes</button>
+                    <a class="btn btn-secondary" href="/admin/teams">Cancel</a>
+                    <button type="button" class="btn btn-danger" @click="deletePrompt">Delete</button>
+                    <em><span v-bind:class="{ 'text-danger': hasError}"> {{feedback}} </span></em>
+                </div>
+
+            </form>
+
+            <ul class="nav nav-tabs">
+                <li class="nav-item">
+                    <a class="nav-link active" id="rsvp-tab" data-toggle="tab" href="#rsvps">Members</a>
+                </li>
+            </ul>
+
+            <div class="tab-content">
+                <div class="tab-pane show active" id="members">
+                    <h3>Members</h3>
+                    <team-invite-modal
+                            id="teamInviteModal"
+                            :teamId="this.teamId">
+                    </team-invite-modal>
+                    <button type="button" class="btn btn-secondary btn-above-table" data-toggle="modal" data-target="#teamInviteModal">Invite</button>
```

Comment:
In most contexts, an Invite is something you have to act on before you are part of a group. This button actually adds people to the team, and so it should be relabeled to indicate that.